pom.xml

```diff
@@ -48,7 +48,7 @@ under the License.
 
     <properties>
         <flink.version>1.16.0</flink.version>
-        <flink.shaded.version>15.0</flink.shaded.version>
+        <flink.shaded.version>16.1</flink.shaded.version>
```

Comment:
We actually should create a follow-up ticket because connectors shouldn't rely on Flink Shaded at all, like we've also done for Cassandra https://issues.apache.org/jira/browse/FLINK-31722